articles/firewall-manager/migrate-to-policy.md

```diff
@@ -110,7 +114,18 @@ If ($azfw.NetworkRuleCollections.Count -gt 0) {
 			Write-Host "creating " $rc.Rules.Count " network rules for collection "  $rc.Name
 			$firewallPolicyNetRules = @()
 			ForEach ($rule in $rc.Rules) {
+				If(($rule.SourceAddresses -and $rule.DestinationAddresses)){
```

Comment:
Please check the indentation. The line after the If block should be one tab inside. Line 113 as a reference

Reply:
Fixed